src/components/Redux/store.js

```diff
@@ -0,0 +1,10 @@
+import { configureStore } from '@reduxjs/toolkit';
+import dataReducer from './data/dataSlice';
+
+const store = configureStore({
+  reducer: {
+    Data: dataReducer,
+  },
+});
```

Comment:
- [ ] It would be better professionally if you put the `Redux` directory inside `src` directly, not inside the `components` folder as you did, So please fix that.

[Hint] The basic project structure of the app suppose to be like that:
```
app-root-directory
  |- package.json
  |- README.md
  |- /public
    |- index.html
  |- /src
    |- app.css
    |- index.js
    |- /components
    |- /redux
  |- /node_modules
```